Fix some issues

b97pla opened the pull request:
- corrects path in bootstrap script
- fix Sarek singularity container name and config

mariya reviewed (changes requested):
This looks good to me. One very minor thing I would add is instructions to copy site_all_env_devel.yml.example to site_all_env_devel.yml in order to get things to run in devel mode. Well done!

b97pla commented:
Thanks! I added a sentence about the example files in the development section. I also added some stuff on using the `site` parameter (although a larger overhaul is needed down the line).

mariya commented:
Approved for merge :)
